### PR TITLE
Create HttpContext

### PR DESCRIPTION
**Pull request by norovferuzbek3963**
HttpContext create

### CODE REVIEW

File: TCBApp/TCBApp.WebApi/HttpContext.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System.Net;
+
+namespace TCBApp.WebApi;
+
+public class HttpContext
+{
+    public WebRequest Request { get; set; }
```
Comment (0605AbMu) [inline TCBApp/TCBApp.WebApi/HttpContext.cs:7]: HttpListenerning type laridan foydalanamiz

**Review by 0605AbMu — CHANGES_REQUESTED** [review]

**Comment by norovferuzbek3963** [comment]
Changes update

**Comment by norovferuzbek3963** [comment]
Update

**Review by 0605AbMu — APPROVED** [review]
Good job!

**Pull request merged**